carla-carma-integration/carla_carma_bridge/src/carla_carma_bridge/carma_to_carla_ackermann_cmd

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2021 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+"""
+Subscribe from CARMA :autoware_msgs::VehicleCmd
+    Topic: /hardware_interface/vehicle_cmd
+
+Subscribe from CARMA :std_msgs::Float32
+    Topic: /guidance/twist_filter/result/twist/lateral_jerk
+
+Subscribe from CARMA :cav_msgs::GuidanceState
+    Topic: /guidance/state
+
+Publish to CARLA :ackermann_msgs::AckermannDrive
+    Topic: /carla/{}/ackermann_cmd
+
+"""
+import rospy
+from ackermann_msgs.msg import AckermannDrive
+from autoware_msgs.msg import VehicleCmd
+from cav_msgs.msg import GuidanceState
+from std_msgs.msg import Float32
+
+ackermann_cmd_pub = None
+guidance_state = GuidanceState.STARTUP
+init_status = True
+init_cmd = AckermannDrive()
+lateral_jerk = 0
+
+def guidance_state_callback(guidance_state_msg):
+    """
+    callback for guidance state subscribing from CARMA
+    guidance_state_msg type:
+        cav_msgs::GuidanceState
+    """
+    global init_status
+    global init_cmd
+    global guidance_state
+    guidance_state = guidance_state_msg.state
+    if guidance_state == GuidanceState.ENGAGED and init_status:
+        init_status = False
+        ## providing initial ackermann command to carla virtual vehicle
+        init_cmd.speed = rospy.get_param('/init_speed', 10)
+        init_cmd.acceleration = rospy.get_param('/init_acceleration', 1)
+        init_cmd.steering_angle = rospy.get_param('/init_steering_angle', 0)
+        init_cmd.jerk = rospy.get_param('/init_jerk', 0)
+        ackermann_cmd_pub.publish(init_cmd)
+
+def commanded_jerk_callback(lateral_jerk_msg):
+    """
+    callback for commanded jerk from CARMA
+    lateral_jerk_msg type:
+        std_msgs::Float32
+    """
+
+    global jerk
```

Comment:
This jerk variable is created (as a reference to a global) but never used here. Either we should delete this line, or perhaps the line after needs a change to reference this variable?

Reply:
Actually, it looks like there's a mismatch, insofaras the global variable exists, but it's called `lateral_jerk` not `jerk`. So I think this line needs an update to match the global variable name.

Reply:
Didn't notice about that, I've updated to match the global variable name